pkg/apprepo/apprepos_handler.go

```diff
@@ -55,9 +54,9 @@ type combinedClientset struct {
 	*kubernetes.Clientset
 }
 
-// appRepositories handles http requests for operating on app repositories
+// AppRepositoriesHandler handles http requests for operating on app repositories
 // in Kubeapps, without exposing implementation details to 3rd party integrations.
-type appRepositoriesHandler struct {
+type AppRepositoriesHandler struct {
```

Comment:
Not sure if this, the conversion from private to public, was done before you created the interface below... I don't see it being accessed outside this module?

Reply:
Apart from that, I was getting a lint warning because the public method NewAppRepositoriesHandler was returning this private struct.